explanations/maintainers-missing.md

```diff
@@ -0,0 +1,3 @@
+All packages ought to have a maintainer.
+
+This can be the update submitter or a community member that accepts to take maintainership of the package. See [Section 7.1 of the Nixpkgs manual](https://nixos.org/manual/nixpkgs/unstable/#sec-standard-meta-attributes) for more details.
```

Comment:
I merged the PR so we will be able to link directly to https://nixos.org/manual/nixpkgs/unstable/#var-meta-maintainers once the manual is rebuilt.